docs/Style-Guide.md

```diff
@@ -0,0 +1,180 @@
+
+One of the main rules of formal writing such as references, synthesis and also in API documentation is to avoid second person ("you"). It should not directly address the user even when the word "you" is almost unavoidable when writing, but there are ways to avoid second person. Below are some alternative examples you can use to evade them. 
+
+**Example**
+
+Less Like this: You can use the following recommendation as an example.
+
+More like this: As an example, The following are recommendations to use.
+
+To view a documented example, visit the [Decorators.md](https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Decorators.md) page on Github. 
+
+
+### Avoid writing condescending languages.
+
+Condescending languages are words that include:
+
+* Just 
+* Easy
+* Simply
+* Basically
+* Obviously
+
+The reader may not find it easy to use the Fastify's framework and plugins, so avoid words that make it sound simple or easy. Not everyone who reads the documentation has the same level of understanding.
+
+
+### Starting with a verb
+
+Mostly start your description with a verb, which makes it simple and precise for the reader to follow. Use present tense more often, it's easier to read and understand than the past or future tense.
+
+**Example**
+
+ Less like this: There is a need for Nodejs to be installed before you can be able to use fastify.
+
+ More like this: Install Nodejs to make use of fastify.
+
+### Grammatical moods 
+
+Grammatical moods is a great way to express your writing, don't want to sound too bossy while making a direct statement. Know when to switch between indicative, imperative, and Subjunctive moods.
+
+
+**Indicative** - Use when making a factual statement or question. 
+
+Example: Since there is no testing framework available, "fastify recommend ways to write tests".
+
+**Imperative** - Use when giving instructions, actions, commands, or when you write your headings.
+
+Example: Install dependencies before starting application development mode.
+
+
+**Subjunctive** -  Use when making suggestions, hypothesis or non-factual statement.
+
+Example: Reading the documentation on our website is recommended to get comprehensive knowledge on the framework plugin.
+
+### Use **active** voice instead of **passive**
+
+Using active voice is a more compact and direct way of conveying your documentation.
+
+**Example**
+
+Passive: The dependencies of the npm package is installed by nodeJs.
+
+Active: NodeJs installs the npm package dependencies.
+
+## Writing Style
+
+### Documentation titles
+
+When creating a new guide, API or reference in the `/docs/` directory, use short titles that best describe the topic of your documentation. 
+Separate each sentence with the dash *(-)* symbol, which makes your `.md` files clear and readable. It also increases the chance of showing up on search engines (Google, DuckDuckg or Bing) in spheres of Search Engine Optimisation.
```

Comment:
`Separate each sentence with the dash *(-)* symbol,`

I have no idea what that means.

Reply:
<img width="273" alt="Screenshot 2020-10-24 at 9 13 10 PM" src="https://user-images.githubusercontent.com/7862178/97092667-bc209480-163d-11eb-8997-63aec34f3fc7.png">

Every documentation in the `/docs/` directory is titled with the dash(-) symbol instead of separating them with spaces.
   
An example was already added to this section.

Since you asked, I think that part wasn't clear enough. I'll rewrite that section to make clear for other readers as well.